src/__tests__/to-have-style.js

```diff
@@ -1,6 +1,7 @@
 import {render} from './helpers/test-utils'
 import document from './helpers/document'
 
+// eslint-disable-next-line max-lines-per-function
```

Comment:
@kentcdodds I've been meaning to raise this issue since I started jest-do: what's the value of this eslint rule? Especially in tests it makes little sense. Asking you because I believe this comes from `kcd-scripts`, isn't it? I know we could go and disable it in our eslint overrides, but maybe this can be disabled at the root.

Reply:
Well, I rarely bump into violations of this rule because I don't use describe blocks. You can feel free to disable it in your overrides, but I think I want to keep it around in my config.

Reply:
What about removing just for specs files? Since we don't have a source analysis tool, this prevents long method bad smells.

Reply:
I'll open a PR for that.